src/App.tsx

```diff
@@ -1,7 +1,13 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+enum Buttons {
```

Comment:
You should also add default value for sorting here and then use it in useState instead of null